Carbon 2185 Quickstart update

wizardeddas opened the pull request:
## Changes / Comments

Added text box for feats/abilities, moved money box up to header, fixed item weight calculation and added room for more items.




## Roll20 Requests

*Include the name of the sheet(s) you made changes to in the title.*

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- Is this a bug fix?
- Does this add functional enhancements (new features or extending existing features) ?
- Does this add or change functional aesthetics (such as layout or color scheme) ? 
- Are you intentionally changing more that one sheet? If so, which ones ?
- If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

Anduh commented:
1. This doesn't look like an update, you're seemingly submitting a new sheet.
2. You have merge conflicts you meed to fix before anything can even be considered for approval.

wizardeddas commented:
1, it is an update, to a sheet I originally created, the Carbon 2185 Quickstart sheet. I moved a couple things around for ease of play, and fixed a bug in the weight calculation for gear.

2, I've resolved the conflicts I believe. Sorry, I'm new to the whole github thing.

Thank you.
